### PR TITLE
Update signing-android.md

### PR DESCRIPTION
**Pull request by K-Codemagic**
Updating Env variable groups - Code Signing Android.

### CODE REVIEW

File: content/yaml-code-signing/signing-android.md [hunk 1]
```diff
@@ -48,15 +48,12 @@ This example shows how to set up code signing using Gradle.
   }
   ...
 ```
-2. Add your keystore and keystore details in the [`environment`](../getting-started/yaml#environment) section of the configuration file. [Encrypt](../building/encrypting/#encrypting-sensitive-data) your keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) and set the encrypted values to the following environment variables. Note that encrypting files requires you to base64 encode them locally and decode them during the build.
+2. Add your keystore and keystore details in the `Environment variables` section of the Codemagic UI. [`Secure`](../building/encrypting/#encrypting-sensitive-data) your keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) and set the values to the following environment variables. Note that securing files requires you to [`base64 encode`](https://docs.codemagic.io/variables/environment-variable-groups/#global-variables-and-secrets) them locally and decode them during the build.
```
Comment (helinanever) [inline content/yaml-code-signing/signing-android.md:51]: We normally use bold to highlight UI elements, so it should be **Environment variables** and **Secure**. Also note that the link for Secure points to a file we no longer have, the right link would be this: ../variables/environment-variable-groups/.
Also we should use relative links everywhere in docs and not full links. 
The base64 encode link currently do not point to the right section either, it should be this: ../variables/environment-variable-groups/#storing-sensitive-valuesfiles. So actually it looks like both links should take to the same section.

Comment (helinanever) [reply content/yaml-code-signing/signing-android.md:51]: I think the whole section should be expanded a bit. The variable names are still important, right? The way I would present the instructions is sth like this:
- Add your environment variables in Codemagic UI (either in application or team variables)
- The names of the variables and what to add there
- A note about encoding binary files and decoding them in yaml
- A note about checking **Secure**
- A note about saving them all to the same group so they can be important to yaml at once

File: content/yaml-code-signing/signing-android.md [hunk 2]
```diff
@@ -90,14 +87,14 @@ The following templates show code signing using `key.properties`.
...
+```
 FCI_KEYSTORE_PATH: /tmp/keystore.keystore
-FCI_KEYSTORE: Encrypted(...)
-FCI_KEYSTORE_PASSWORD: Encrypted(...)
-FCI_KEY_ALIAS: Encrypted(...)
-FCI_KEY_PASSWORD: Encrypted(...)
+FCI_KEYSTORE
+FCI_KEYSTORE_PASSWORD
+FCI_KEY_ALIAS
+FCI_KEY_PASSWORD
```
Comment (helinanever) [inline content/yaml-code-signing/signing-android.md:97]: These variables need to be added in the UI as well and not in yaml

**Comment by K-Codemagic** [comment]
Changes have been made, let me know if any more updation is required, so that can be useful in other raised PRs, to follow

File: content/yaml-code-signing/signing-android.md [hunk 3]
```diff
@@ -48,16 +48,24 @@ This example shows how to set up code signing using Gradle.
   }
   ...
 ```
-2. Add your keystore and keystore details in the [`environment`](../getting-started/yaml#environment) section of the configuration file. [Encrypt](../building/encrypting/#encrypting-sensitive-data) your keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) and set the encrypted values to the following environment variables. Note that encrypting files requires you to base64 encode them locally and decode them during the build.
+2. Save the keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) to the respective environment variables in the **Environment variables** section in Codemagic UI. Click **Secure** to encrypt the values. Note that binary files i.e. keystore file requires you to [`base64 encode`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) them locally, before they can be saved to environment variables and decode them during the build.
```
Comment (helinanever) [inline content/yaml-code-signing/signing-android.md:51]:
```suggestion
2. Save the keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) to the respective environment variables in the **Environment variables** section in Codemagic UI. Click **Secure** to encrypt the values. Note that binary files (i.e. keystore) have to be [`base64 encoded`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) locally before they can be saved to environment variables and decoded during the build.
```

File: content/yaml-code-signing/signing-android.md [hunk 4]
```diff
@@ -90,16 +98,21 @@ The following templates show code signing using `key.properties`.
 ```
 ### Set up code signing with user-specified keys
 
-In order to do code signing [encrypt](../building/encrypting/#encrypting-sensitive-data) your keystore file, keystore password (if keystore is password protected), key alias and key alias password (if key alias is password protected) and set the encrypted values to the following environment variables:
+In order to do code signing Save the keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) to the respective environment variables in the **Environment variables** section in Codemagic UI. Click **Secure** to encrypt the values. Note that binary files i.e. keystore file requires you to [`base64 encode`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) them locally, before they can be saved to environment variables and decode them during the build:
```
Comment (helinanever) [inline content/yaml-code-signing/signing-android.md:101]:
```suggestion
In order to code sign the build, save the keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) to the respective environment variables in the **Environment variables** section in Codemagic UI. Click **Secure** to encrypt the values. Note that binary files (i.e. keystore) have to be [`base64 encode`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) locally before they can be saved to environment variables and decoded during the build:
```

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**